src/components/Form/CrossPodWriteForm.jsx

```diff
@@ -85,66 +98,102 @@ const CrossPodWriteForm = () => {
+          <FormControl fullWidth>
+            <Typography>Upload document to username:</Typography>
+            <TextField
+              name="crossPodUpload"
+              label="Search Username"
+              required
+              value={selectedUser.person ? selectedUser.person : username}
+              onChange={(e) => setUsername(e.target.value)}
+              placeholder={selectedUser.username}
+            />
+          </FormControl>
           <DocumentSelection
             htmlForAndIdProp="upload-doc"
             handleDocType={handleDocType}
             docType={docType}
           />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="upload-doc-expiration">Expiration date (if applicable): </label>
-          <input id="upload-doc-expiration" name="date" type="date" />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="upload-doc-desc">Enter description: </label>
           <br />
+          <FormControl>
+            <LocalizationProvider dateAdapter={AdapterDayjs}>
+              <DatePicker
+                name="date"
+                format="MM/DD/YYYY"
+                label="Expiration Date"
+                value={expireDate}
+                onChange={(newExpireDate) => setExpireDate(newExpireDate)}
+                type="date"
+              />
+              <FormHelperText>MM/DD/YYYY</FormHelperText>
+            </LocalizationProvider>
+          </FormControl>
           <br />
-          <textarea id="upload-doc-desc" name="description" {...description} />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="upload-doctype">File to upload: </label>
-          <input
-            id="upload-doctype"
-            type="file"
-            name="uploadDoctype"
-            accept=".pdf, .docx, .doc, .txt, .rtf, .gif"
-            onChange={handleFileChange}
-          />
-          <button disabled={state.processing} type="submit">
-            Upload file
-          </button>
-        </div>
-      </form>
+          <br />
+          <FormControl fullWidth>
+            <TextField
+              name="description"
+              multiline
+              rows={4}
+              label="Enter Description"
+              value={docDescription}
+              onChange={(newDocDescription) => setDocDescription(newDocDescription.target.value)}
+              placeholder="Add a description here"
+            />
+          </FormControl>
+          <br />
+          <br />
+          <FormControl fullWidth>
+            <Button
+              variant="contained"
+              component="label"
+              color="primary"
+              id="upload-doctype"
+              name="uploadDoctype"
+              onChange={handleFileChange}
+              required
+            >
+              Choose file
+              <input type="file" hidden accept=".pdf, .docx, .doc, .txt, .rtf" multiple />
```

Comment:
The `accept` attribute should include .gif like before. As for `multiple`, we don't have functionality to support that at the moment so we can remove that for now.